stable/jenkins/README.md

```diff
@@ -68,7 +68,10 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `master.tag`                      | Master image tag                     | `lts`                                     |
 | `master.imagePullPolicy`          | Master image pull policy             | `Always`                                  |
 | `master.imagePullSecretName`      | Master image pull secret             | Not set                                   |
+| `master.disableRememberMe`        | Diable use of remember me            | `false`                                   |
+| `master.mode`                     | Jenkins mode (NORMAL or EXCLUSIVE)   | `NORMAL`                                  |
 | `master.numExecutors`             | Set Number of executors              | 0                                         |
+| `master.defaultMarkupFormatter`   | Use default markupFormatter          | `true`                                    |
```

Comment:
I'm not understanding the purpose of this field

Reply:
I use a markup formatter that isn't plaintext or safe html.  Rather than add another fork and about ten new values to the chart I thought if that part of the config could just be removed that would be a way for users of the chart to configure the markup formatter in an additional jcasc file.  Does that make sense?

Reply:
yeah, this is definitely tricky. since you're using jcasc, how about adding `master.markupFormatter` for setting the jcasc. I think it would be ok not to support this feature in the XML configuration as we're working on a v2 of the chart in which jcasc will be the default. I ran a few tests with the following and it seems to work.

**values.yaml**
```yaml
master:
  markupFormatter: plainText
```
**_helpers.tpl**
```yaml
  markupFormatter:
    {{- if .Values.master.enableRawHtmlMarkupFormatter }}
    rawHtml:
      disableSyntaxHighlighting: true
    {{- else }}
    {{- toYaml .Values.master.markupFormatter | nindent 4 }}
    {{- end }}
```